### PR TITLE
feat: add boost.bind

### PR DESCRIPTION
**Pull request by wep21**
- add [boost.bind](http://boost.org/libs/bind)
- need skip url stability check

### CODE REVIEW

**Comment by bazel-io** [comment]
Hello @bazelbuild/bcr-maintainers, modules without existing maintainers (boost.bind, boost.detail) have been updated in this PR. Please review the changes.

**Comment by bazel-io** [comment]
Hello @bazelbuild/bcr-maintainers, modules without existing maintainers (boost.bind) have been updated in this PR. Please review the changes.

File: modules/boost.bind/1.83.0/MODULE.bazel [hunk 1]
```diff
@@ -0,0 +1,9 @@
+module(
+    name = "boost.bind",
+    version = "1.83.0",
```
Comment (meteorcloudy) [inline modules/boost.bind/1.83.0/MODULE.bazel:3]: Please also add bazel_compatibility > 7.2.1 like the other PR.

**Comment by wep21** [comment]
@meteorcloudy @fmeum @Vertexwahn I cannot reproduce the error on ci in my environment. Is it possible to reproduce the ci environment?

**Comment by meteorcloudy** [comment]
I can reproduce with 
```
bazel build @boost.bind//:boost.bind --registry=https://raw.githubusercontent.com/wep21/bazel-central-registry/add-boost.bind --process_headers_in_dependencies
```

**Comment by wep21** [comment]
In my environment
```
❯ bazel build @boost.bind//:boost.bind --registry=https://raw.githubusercontent.com/wep21/bazel-central-registry/add-boost.bind --process_headers_in_dependencies
Starting local Bazel server and connecting to it...
INFO: Analyzed target //:boost.bind (92 packages loaded, 633 targets configured).
INFO: From Compiling include/boost/bind.hpp:
include/boost/bind.hpp:36:1: warning: The practice of declaring the Bind placeholders (_1, _2, ...) in the global namespace is deprecated. Please use <boost/bind/bind.hpp> + using namespace boost::placeholders, or define BOOST_BIND_GLOBAL_PLACEHOLDERS to retain the current behavior. [-W#pragma-messages]
BOOST_PRAGMA_MESSAGE(
^
external/boost.config~/include/boost/config/pragma_message.hpp:24:34: note: expanded from macro 'BOOST_PRAGMA_MESSAGE'
# define BOOST_PRAGMA_MESSAGE(x) _Pragma(BOOST_STRINGIZE(message(x)))
                                 ^
<scratch space>:15:2: note: expanded from here
 message("The practice of declaring the Bind placeholders (_1, _2, ...) " "in the global namespace is deprecated. Please use " "<boost/bind/bind.hpp> + using namespace boost::placeholders, " "or define BOOST_BIND_GLOBAL_PLACEHOLDERS to retain the current behavior.")
 ^
1 warning generated.
INFO: Found 1 target...
Target //:boost.bind up-to-date (nothing to build)
INFO: Elapsed time: 10.269s, Critical Path: 0.41s
INFO: 12 processes: 1 internal, 11 darwin-sandbox.
INFO: Build completed successfully, 12 total actions
```

**Comment by meteorcloudy** [comment]
Hmm, maybe you can try with one of the CI docker image `gcr.io/bazel-public/ubuntu2204`

**Comment by wep21** [comment]
@meteorcloudy I'm sorry, but I see the different version of boost.bind.

**Comment by wep21** [comment]
@meteorcloudy Yes, I created a BUILD file based on `v1.85.0`, so I changed file names to fit `v1.83.0`.

**Review by meteorcloudy — APPROVED** [review]
Thanks!

**Comment by meteorcloudy** [comment]
@wep21 Just in case you didn't know, you can submit multiple modules in one PR. Feel free to combine a bunch of simple ones to accelerate the merge process ;)

**Pull request merged**